### ISSUE TITLE
ERR_REQUIRE_ESM occurs when just invoke `fontmin`

### ISSUE THREAD

**Issue by aztack** [ecomfe/fontmin#111]
Error occurs when just invoke `fontmin`:
Node 16.15.1

```bash
/Users/username/.config/yarn/global/node_modules/fontmin/cli.js:13
var meow = require('meow');
           ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/username/.config/yarn/global/node_modules/fontmin/node_modules/meow/index.js from /Users/username/.config/yarn/global/node_modules/fontmin/cli.js not supported.
Instead change the require of index.js in /Users/username/.config/yarn/global/node_modules/fontmin/cli.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/username/.config/yarn/global/node_modules/fontmin/cli.js:13:12) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by qiutian00** [comment]
also come this propblem

**Comment by kekee000** [comment]
fixed in fontmin@1.1.0
